.eslintrc

```diff
@@ -1,6 +1,5 @@
 {
   "root": true,
-  "ecmaVersion": 6,
```

Comment:
We should still keep that. Not really related to normal updates as far as I can see.

Reply:
The updated version of eslint moves the inside of `parseOptions`.